### PR TITLE
Add retry to get elastictest token

### PR DESCRIPTION
**Pull request by xwjiang-ms**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [x] Testbed and Framework(new/improvement)
- [ ] Test case(new/improvement)

### Back port request
- [ ] 201911
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311

#### What is the motivation for this PR?
There is no retry to get elastictest token, if get token fails, whole build would fail
#### How did you do it?
Add a retry to get elastictest token
Since kvm dump is no longer supported, set default value to false

### CODE REVIEW

File: .azure-pipelines/test_plan.py [hunk 1]
```diff
@@ -188,13 +188,18 @@ def get_token(self):
...
         }
-        try:
-            resp = requests.post(token_url, headers=headers, data=payload, timeout=10).json()
-            self._token = resp["access_token"]
-            self._token_generate_time = datetime.utcnow()
-            return self._token
-        except Exception as exception:
-            raise Exception("Get token failed with exception: {}".format(repr(exception)))
+        max_attempts = 3
+        for attempt in range(max_attempts):
```
Comment (yejianquan) [inline .azure-pipelines/test_plan.py:192]: If we increase attempt on line 201, the loop here should be while(attempt < max_attpemts), right?

**Review by yejianquan — CHANGES_REQUESTED** [review]

File: .azure-pipelines/test_plan.py (hunk 1 above)
Comment (xwjiang-ms) [reply .azure-pipelines/test_plan.py]: Fixed

**Review by yejianquan — APPROVED** [review]
LGTM

**Pull request merged**